lib/gooddata/lcm/actions/base_action.rb

```diff
@@ -31,6 +36,134 @@ def without_check(specification, params)
           params.setup_filters(specification) # enables params validation
           result
         end
+
+        def print_result(_params)
+          true
+        end
+
+        def allow_warning(params)
+          Helpers.allow_warning(params)
+        end
+
+        def add_project_warning_message(project_id, message, error_action, params)
+          running_status = warning_status(params)
```

Comment:
it's hard to understand running_status=warning_status, please rename warning_status, running_status to self-descriptive

Reply:
Done update code